source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2174,37 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, PerformBridgeShuntCalEx_Succeeds)
```

Comment:
No, it doesn't succeed. Update the test name and add a brief comment explaining why this is an error test.